docs/Measuring_Resonances.md

```diff
@@ -65,45 +41,116 @@ the system that may damage the electronics.
 
-Afterwards, check and follow the instructions in the
-[RPi Microcontroller document](RPi_microcontroller.md) to setup the
-"linux mcu" on the Raspberry Pi.
+Double-check your wiring before powering up the Raspberry Pi to prevent
+damaging it or the accelerometer.
 
-Make sure the Linux SPI driver is enabled by running `sudo
-raspi-config` and enabling SPI under the "Interfacing options" menu.
+Check and follow the
+instructions in the [RPi Microcontroller document](RPi_microcontroller.md) to setup the
+"linux mcu" on the Raspberry Pi. Make sure the Linux SPI driver is enabled by running
+`sudo raspi-config` and enabling SPI under the "Interfacing options" menu.
 
 Add the following to the printer.cfg file:
+
 ```
 [mcu rpi]
 serial: /tmp/klipper_host_mcu
 
 [adxl345]
-cs_pin: rpi:None
+cs_pin: rpi: None
 
 [resonance_tester]
 accel_chip: adxl345
 probe_points:
-    100,100,20  # an example
+    100,100,20  # It is advised to start with 1 probe point, in the middle
+    of the print bed, slightly above it.
 ```
-It is advised to start with 1 probe point, in the middle of the print bed,
-slightly above it.
+
+Restart Klipper via the `RESTART` command.
+
+## Wiring Option 2: MCU Connection (Advanced Users)
+If you want to avoid installing the Linux MCU software on the Raspberry Pi, your Pi's
+GPIO is already in use, or some other reason prevents you from using option 1, you can
+connect the accelerometer to your Primary MCU, or even a secondary MCU. There are some
+caveats, however:
+* As most printer boards output 5V power for accessories, you'll likely need an ADXL345
+board with a built in voltage level shifter.
+* The ADXL345 does not adhere to the SPI spec and cannot share an SPI bus with another
+device. (Thanks @dmbutyugin for pointing that out!) The sensor will need to be the only
+device connected to its SPI bus.
+* The sensor requires a reasonably fast SPI bus
+
+Identify the MOSI, MISO, SCLK, CS, Ground, and power (see above) pins to be used for
+your sensor anc connect your sensor as follows:
+
+| ADXL345 pin | MCU |
+| :--:        | :--: |
+| 3V3/5V/VCC  | 3V3/5V/VCC |
+| GND         | Ground |
+| CS          | Any |
+| SDO         | MISO |
+| SDA         | MOSI |
+| SCL         | SCLK |
+
+Double-check your wiring, especially the voltage tolerated by your sensor and that
+provided by it's connection, before powering up your printer to prevent damaging it
+or the accelerometer.
+
+Add something like the following to the printer.cfg file:
+
+```
+[adxl345]
+cs_pin: AB0 # The MCU pin connected to the CS pin on your sensor
+spi_bus: # This may be necessary if your board has multiple busses available
+
+[resonance_tester]
+accel_chip: adxl345
+probe_points:
+    100,100,20  # It is advised to start with 1 probe point, in the middle
+    of the print bed, slightly above it.
+```
+
+For more information about how to configure the SPI behavior of Klipper, review the
+[common SPI settings](config_reference.md#common_SPI_settings).
```

Comment:
I believe this link and the one below don't work. I think the right references are Config_Reference.md#common-spi-settings and Config_Reference.md#mcu-my_extra_mcu